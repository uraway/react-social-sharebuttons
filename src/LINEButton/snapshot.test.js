import React from 'react';
import renderer from 'react-test-renderer';
import LineButton from '.';

it('renders correctly with required props', () => {
  const tree = renderer.create(<LineButton />).toJSON();
  expect(tree).toMatchSnapshot();
});

it('renders correctly with other props', () => {
  const tree = renderer.create(<LineButton text="text" image="82x20" />).toJSON();
  expect(tree).toMatchSnapshot();
});
