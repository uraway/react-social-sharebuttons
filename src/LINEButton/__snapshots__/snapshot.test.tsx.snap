// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with other props 1`] = `
<div>
  <a
    href="http://line.me/R/msg/text/?text"
  >
    <img
      alt="LINEで送る"
      src="http://i.imgur.com/cfjCxrh.png"
      style={
        Object {
          "height": 20,
          "width": 82,
        }
      }
    />
  </a>
</div>
`;

exports[`renders correctly with required props 1`] = `
<div>
  <a
    href="http://line.me/R/msg/text/?"
  >
    <img
      alt="LINEで送る"
      src="http://i.imgur.com/voMN0NH.png"
      style={
        Object {
          "height": 20,
          "width": 20,
        }
      }
    />
  </a>
</div>
`;
